.eslintrc

```diff
@@ -9,5 +9,6 @@
   "rules": {
     "no-mixed-operators": [2, { "allowSamePrecedence": true }],
     "jsx-a11y/no-static-element-interactions": 1, // TODO: enable
+    "react/no-did-mount-set-state": 0, // necessary for server-rendering
```

Comment:
Can we exclusively put this on the `DateInput` instead of as a universal exception?